internal/drivers/postgresql/postgresql.go

```diff
@@ -126,6 +128,13 @@ func (p *postgresqlDriver) Flush() error {
 		}()
 		if _, err := tx.ExecContext(p.ctx, p.pending.String()); err != nil {
 			p.logger.Trace("offending sql: %s", p.pending.String())
+			//Post the offending sql to a file log. For large batch inserts, the offending SQL takes up too much space in the terminal
```

Comment:
//                          spaaaaaaaaaaaaace after slashesssssssss
```suggestion
			// Post the offending sql to a file log. For large batch inserts, the offending SQL takes up too much space in the terminal
```